courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<ul><li><a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/3f8f177875ebf0d3e70c0c4d8affcb1dbf09275e/courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb" target="_blank">BQML Baseline Model</a> is broken</li><li>Remove "In this lab" </li><li>Since this is the solution, remove: "Each learning objective will correspond to a&nbsp;<strong>#TODO</strong>&nbsp;in this student lab notebook -- try to complete this notebook first and then review the&nbsp;<a href="https://github.com/GoogleCloudPlatform/training-data-analyst/raw/3f8f177875ebf0d3e70c0c4d8affcb1dbf09275e/courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb" target="_blank">solution notebook</a>."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Done</p>